SignalServiceKit/src/Messages/DeviceSyncing/OWSOutgoingSentMessageTranscript.m

```diff
@@ -39,6 +42,8 @@ - (instancetype)initWithOutgoingMessage:(TSOutgoingMessage *)message
     }
 
     _message = message;
+    // This will be nil for groups.
+    _sentRecipientId = message.thread.contactIdentifier;
```

Comment:
I guess this is tricky, because we want it to be nil for group threads... we could pass in a transaction and do some more work, but we've had this logic for a long time, it just got shuffled a lot recently (starting with 9275c67818d4d7f98b82d193c60d8fca9c4208b0). 

Since we're effectively back to what we originally had with this field (though now we're caching it which is probably good), I'm fine with this change as is.